doc/examples/reconst_mapmri.py

```diff
@@ -32,35 +47,315 @@
 """
 For this example we select only the shell with b-values equal to the one of the
 Human Connectome Project (HCP).
+
+data contains the voxel data and gtab contains a GradientTable
+object (gradient information e.g. b-values). For example, to show the b-values
+it is possible to write print(gtab.bvals).
+
+For the values of the q-space
+indices to make sense it is necessary to explicitly state the big_delta and
+small_delta parameters in the gradient table.
 """
+
 bvals = [1000, 2000, 3000]
 img, gtab = read_cenir_multib(bvals)
+big_delta = 0.0365  # seconds
+small_delta = 0.0157  # seconds
+gtab = gradient_table(bvals=gtab.bvals, bvecs=gtab.bvecs,
+                      small_delta=big_delta,
+                      big_delta=small_delta)
 data = img.get_data()
 data_small = data[40:65, 50:51, 35:60]
 
 print('data.shape (%d, %d, %d, %d)' % data.shape)
 
 """
-data contains the voxel data and gtab contains a GradientTable
-object (gradient information e.g. b-values). For example, to show the b-values
-it is possible to write print(gtab.bvals).
+The MAPMRI Model can now be instantiated. The radial_ordermdetermines the
```

Comment:
"The radial_orderdetermines..."=> "The ``radial_order``  parameter determines..."